api/v2/README.md

```diff
@@ -17,7 +17,7 @@ For details on using the API, see the following pages.
 | [Encrypting Requests and Decrypting Responses](ref-info/encryption-decryption.md) | The high-level request-response workflow for the UID2 APIs, requirements for encrypting requests and decrypting responses, and respective script examples in Python.  |
 | [Endpoints](./endpoints/README.md) | The API reference for managing identity tokens and mapping email addresses, phone numbers, or hashes to their UID2s and salt bucket IDs used to generate the UID2s. |
 | [Integration Guides](./guides/README.md) | The UID2 integration workflows for UID2 participants, such as publishers, DSPs, advertisers, and data providers, as well as Operator Enterprise Partners, such as Microsoft Azure, AWS, and Snowflake. |
-| [SDKs](./sdks/README.md) | The Client-Side JavaScript for websites and RTB SDKs. | 
+| [SDKs](./sdks/README.md) | Links to SDK docs, including the Client-Side JavaScript SDK for websites and the Server-Side SDK Guide for RTB. | 
```

Comment:
The Java SDK also has support for Publishers (not just RTB/DSPs). Should we mention that in these pages?

Reply:
Yes -- thanks. For now, until I get this SDK section filled out accurately, I just cut out the last bit. "Links to SDK docs."